### ISSUE TITLE
error when using 3 button navigation

### ISSUE THREAD

**Issue by pbzinwindows** [RisingOS-Revived/android_build_soong#1]
When I select navigation gestures, if I try to go back to using 3 buttons when clicking on the option, it automatically selects gestures again

**Comment by pbzinwindows** [comment]
option to activate accessibility apps also does not work
